client/my-sites/domains/domain-management/components/domain/domain-info-card/email/types.tsx

```diff
@@ -1,9 +0,0 @@
-export type Mailbox = {
-	domain: string;
-	mailbox: string;
-};
-
-export type EmailAccount = {
-	account_type: string;
-	emails: Mailbox[];
-};
```

Comment:
Removing this one because there's absolutely no usage. Confirmed with @rafaelgallani